intelmq/bots/parsers/fireeye/parser.py

```diff
@@ -0,0 +1,92 @@
+                                            # context inhalt
+                                            if context_search[0] == "Network/HTTP/RequestURI":
+                                                classification = "destination.urlpath"
+                                            if context_search[0] == "Network/HTTP/Host":
+                                                classification = "destination.ip"
+                                            if context_search[0] == "Network/Connection/RemotePort":
+                                                classification = "destination.port"
+                                            if context_search[0] == "Network/Connection/Protocol":
+                                                classification = "protocol.transport"
+                                            if context_search[0] == "Network/DNS":
+                                                classification = "destination.fqdn"
+                                        if 'Content' in searchIndicator:
+                                            Content_search = searchIndicator.split('">')
+                                            context = Content_search[1].split('</Content>')
+                                            self.logger.debug(classification + "   "+context[0])
+                                            if fqdn != "" and urlpath != "":
+                                                event.add("destination.url", "http://" + fqdn+urlpath)
+                                                fqdn = ""
+                                                urlpath = ""
+                                            if classification == "destination.ip":
+                                                try:
+                                                    ipaddress.IPv4Network(context[0])
+                                                    event.add('destination.ip', context[0])
+                                                    break
+                                                except ValueError:
+                                                    break
+                                            elif classification == "destination.fqdn":
+                                                fqdn = context[0]
+                                                event.add(classification, context[0])
+                                            elif classification == "destination.urlpath":
+                                                urlpath = context[0]
+                                                event.add(classification, context[0])
+                                            elif classification == "destination.port":
+                                                event.add(classification, int(context[0]))
+                    self.send_message(event)
+            self.acknowledge_message()
+        except KeyError:
+            self.logger.info("No Iocs Available")
```

Comment:
does that really apply to *any* KeyError in this huge block? Catching too wide errors can hide other errors, including programming errors and (unexpected) differences in the data format